packages/client/src/DisplayContent.tsx

```diff
@@ -92,19 +104,43 @@ function displayMarkdown(text: string, props: object) {
   return <div {...props} dangerouslySetInnerHTML={out} />
 }
 
+function displayAttachment(
+  filename: string | null | undefined,
+  subtype: string,
+  props: object
+) {
+  const name = filename ? (
+    <div {...props}>{filename}</div>
+  ) : (
+    <div {...props}>[Attachment.{subtype}}</div>
```

Comment:
Very very small fix, but you have a `}` closing the string instead of matching the `[` at the beginning.